src/src/Helpers/Modbus_link.cpp

```diff
@@ -279,34 +258,70 @@ void ModbusLINK_struct::processCommand()
         it++;
         break;
     } // switch
-    addLogMove(LOG_LEVEL_INFO, log);
   }   // next iterarion
 
   return;
 }
 
 void ModbusLINK_struct::dumpQueueElement(Modbus_RequestQueueElement *el) {
-  String log = F("[ ID=");
-
-  log += el->_id;
-  log += F(", Device=");
-  log += String((uint32_t)(el->_device), HEX);
-  log += F(", State=");
-  log += (uint)el->_state;
-  log += F(", TX=");
-
-  for (int i = 0; i < el->_sendframe_length; i++) {
-    log += String(el->_sendframe[i], HEX);
-    log += F(",");
+  # ifdef MODBUS_DEBUG
+
+  if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
+    String log = F("[ ID=");
+
+    log += el->_id;
+    log += F(", Device=");
+    log += String((uint32_t)(el->_device), HEX);
+    log += F(", State=");
```

Comment:
Here, `strformat()` can also help by using the hex formatting via `%x`, instead of first creating another String